client/layout/sidebar/menu.jsx

```diff
@@ -0,0 +1,10 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+const SidebarMenu = ( { children } ) => (
+	<li className="sidebar__menu">{ children }</li>
```

Comment:
I notice that most all usages of `<SidebarMenu />` render a `<ul>` and optionally a header. I wonder if we could make this a bit easier by removing the concern for rendering the list, i.e.:

```jsx
// In usage:
<SidebarMenu heading={ this.translate( 'My Section' ) }>
	<MyMenuItem />
	<MyMenuItem />
</SidebarMenu>

// Implemented as:
const SidebarMenu = ( { heading, children } ) => (
	<li className="sidebar__menu">
		{ heading && <SidebarHeading>{ heading }</SidebarHeading> }
		<ul>
			{ React.Children.map( children, ( child ) => {
				<li>{ child }</li>
			} ) }
		</ul>
	</li>
);
```